Documentation/Extensions/SceneTransitionService/SceneTransitionServiceOverview.md

```diff
@@ -0,0 +1,187 @@
+
+## Profile Options
+### Use Default Progress Indicator
+If checked, the default progress indicator prefab will be used when no progress indicator object is provided when calling `DoSceneTransition.` If a progress indicator object is provided, the default will be ignored.
+
+### Use Fade Color
+If checked, the transition service will apply a fade during your transition. This setting can be changed at runtime via the service's `UseFadeColor` property.
+
+### Fade Color
+Controls the color of the fade effect. Alpha is ingored. This setting can be changed at runtime prior to a transition via the service's `FadeColor` property.
+
+### Fade Targets
+Controls which cameras will have a fade effect applied to them. This setting can be changed at runtime via the service's `FadeTargets` property.
+Setting | Targeted Cameras
+--- | --- | ---
+Main | Applies fade effect to the main camera.
+UI | Applies fade effect to cameras on the UI layer. (Does not affect overlay UI)
+All | Applies to both main and UI cameras.
+Custom | Applies to a custom set of cameras provided via `SetCustomFadeTargetCameras`
+
+### Fade Out Time / Fade In Time
+Default settings for the duration of a fade on entering / exiting a transition. These settings can be changed at runtime via the service's `FadeOutTime` and `FadeInTime` properties.
+
+### Camera Fader Type
+Which `ICameraFader` class to use for applying a fade effect to cameras. The default `CameraFaderQuad` class instantiates a quad with a transparent material in front of the target camera close to the clip plane. Another approach might be to use a post effects system.
+
+# Using the Extension
+
+You use the transition service by passing Tasks that are run while the camera is faded out.
+
+## Using Scene System Tasks
+
+In most cases you will be using Tasks supplied by the SceneSystem service:
+
+```
+private async void TransitionToScene()
+{
+    IMixedRealitySceneSystem sceneSystem = MixedRealityToolkit.Instance.GetService<IMixedRealitySceneSystem>();
+    ISceneTransitionService transition = MixedRealityToolkit.Instance.GetService<ISceneTransitionService>();
+
+    // Fades out
+    // Runs LoadContent task
+    // Fades back in
+    await transition.DoSceneTransition(
+            () => sceneSystem.LoadContent("TestScene1")
+        );
+}
+```
+
+## Using Custom Tasks
+In other cases you may want to perform a transition without actually loading a scene:
+
+```
+private async void TransitionToScene()
+{
+    ISceneTransitionService transition = MixedRealityToolkit.Instance.GetService<ISceneTransitionService>();
+
+    // Fades out
+    // Resets scene
+    // Fades back in
+    await transition.DoSceneTransition(
+            () => ResetScene()
+        );
+}
+
+private async Task ResetScene() 
+{
+    // Go through all enemies in the current scene and move them back to starting positions   
+}
+```
+
+Or you may want to load a scene without using the SceneSystem service:
```

Comment:
Do you have any guidance on when you would recommend doing one of these other two locations, or is this something that is obvious to people who have done a lot of work in this space?

Reply:
It should be pretty obvious, they'll either be using Unity's scene manager or their own custom wrapper.